src/state/zustand/swapperStore/testData.ts

```diff
@@ -0,0 +1,129 @@
+            proportion: '1',
+          },
+        ],
+        allowanceContract: '0x0000000000000000000000000000000000000000',
+        buyAsset: FOX,
+        sellAsset: ETH,
+        accountNumber: 0,
+      },
+      inputOutputRatio: 0.8560784948911982,
+    },
+    {
+      swapper: {
+        name: SwapperName.Thorchain,
+      } as Swapper<ChainId>,
+      quote: {
+        rate: '52867.94647736405036163943',
+        maximum: '100000000000000000000000000',
+        sellAmountBeforeFeesCryptoBaseUnit: '18665000000000000',
+        buyAmountCryptoBaseUnit: '986780221000000000000',
+        sources: [
+          {
+            name: 'THORChain',
+            proportion: '1',
+          },
+        ],
+        buyAsset: FOX,
+        sellAsset: ETH,
+        accountNumber: 0,
+        minimumCryptoHuman: '0.00576',
+        recommendedSlippage: '0.00009729507077504896',
+        allowanceContract: '0xD37BbE5744D730a1d98d8DC97c42F0Ca46aD7146',
+        feeData: {
+          networkFeeCryptoBaseUnit: '1800000000000000',
+          buyAssetTradeFeeUsd: '8.5196229516043797526224',
+          sellAssetTradeFeeUsd: '0',
+        },
+      },
+      inputOutputRatio: 0.6773207214158627,
+    },
+  ],
+  activeSwapperWithMetadata: {
+    swapper: {
+      name: SwapperName.Thorchain,
+    } as Swapper<ChainId>,
+    quote: {
+      rate: '52867.94647736405036163943',
+      maximum: '100000000000000000000000000',
+      sellAmountBeforeFeesCryptoBaseUnit: '18665000000000000',
+      buyAmountCryptoBaseUnit: '986780221000000000000',
+      sources: [
+        {
+          name: 'THORChain',
+          proportion: '1',
+        },
+      ],
+      buyAsset: FOX,
+      sellAsset: ETH,
+      accountNumber: 0,
+      minimumCryptoHuman: '0.00576',
+      recommendedSlippage: '0.00009729507077504896',
+      allowanceContract: '0xD37BbE5744D730a1d98d8DC97c42F0Ca46aD7146',
+      feeData: {
+        networkFeeCryptoBaseUnit: '1800000000000000',
+        buyAssetTradeFeeUsd: '8.5196229516043797526224',
+        sellAssetTradeFeeUsd: '0',
+      },
+    },
+    inputOutputRatio: 0.6773207214158627,
+  },
+  buyAsset: FOX,
+  sellAsset: ETH,
+  sellAssetFiatRate: '1767.99457108211749084991',
+  buyAssetFiatRate: '0.03336135366491256374',
+  feeAssetFiatRate: '1767.99457108211749084991',
+  receiveAddress: '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6',
+  selectedSellAssetAccountId: 'eip155:1:0x32dbc9cf9e8fbcebe1e0a2ecf05ed86ca3096cb6',
+  selectedBuyAssetAccountId: 'eip155:1:0x32dbc9cf9e8fbcebe1e0a2ecf05ed86ca3096cb6',
+  fees: {
+    chainSpecific: {
+      approvalFeeCryptoBaseUnit: '1800000000000000',
+      gasPriceCryptoBaseUnit: '18000000000',
+      estimatedGas: '100000',
+      totalFee: '0.0036',
+    },
+    tradeFeeSource: 'THORChain',
+    buyAssetTradeFeeUsd: '8.5196229516043797526224',
+    sellAssetTradeFeeUsd: '0',
+    networkFeeCryptoHuman: '0.0018',
+    networkFeeCryptoBaseUnit: '1800000000000000',
+  },
+} as SwapperState
```

Comment:
unnecessary cast?

Reply:
Ish - it was to avoid mocking the swapper store actions.

Mocked and removed cast in https://github.com/shapeshift/web/pull/4138/commits/5179db979e52c13eb6e7ed0b5222a0d6aa67993d.